homeassistant/components/drop_connect/coordinator.py

```diff
@@ -34,11 +34,22 @@ async def set_water(self, value: int):
             payload,
         )
 
-    async def set_bypass(self, value: int):
+    async def set_bypass(self, value: int) -> None:
         """Change water bypass state."""
         payload = self.drop_api.set_bypass_message(value)
         await mqtt.async_publish(
             self.hass,
             self.config_entry.data[CONF_COMMAND_TOPIC],
             payload,
         )
+
+    async def set_protect_mode(self, value: str) -> None:
+        """Change protect mode state."""
+        payload = self.drop_api.set_protect_mode_message(value)
+        await mqtt.async_publish(
+            self.hass,
+            self.config_entry.data[CONF_COMMAND_TOPIC],
+            payload,
+            qos=0,
+            retain=False,
```

Comment:
Like you did for the other methods you can omit passing the default values.

```suggestion
```

Reply:
Ah, yes.  Thanks for noticing that.  Fixed in b5bba9d5f87109e9237337130cf5f4eb96986907.